Sources/HaCWebsiteLib/Events/Event.swift

```diff
@@ -0,0 +1,14 @@
+import Foundation
+
+protocol Event {
+    var title : String {get}
+    var time : Date {get}
+    var eventDescription: String {get} //NOT related to event content (e.g. workshop content)
+    var colour : String {get}
+    var hypeDays : Int {get}
+    var ttlDays : Int {get}
```

Comment:
Thinking about the best way to do this one, but would the range between `hypeDays` and `ttlDays` be better expressed as a [DateInterval](https://developer.apple.com/documentation/foundation/dateinterval)?

Reply:
Had no idea this datatype existed - that's going to make it soooo much nicer ;)